Firestore/core/src/remote/serializer.cc

```diff
@@ -1128,15 +1131,17 @@ google_firestore_v1_Cursor Serializer::EncodeBound(const Bound& bound) const {
   return result;
 }
 
-Bound Serializer::DecodeBound(google_firestore_v1_Cursor& cursor) const {
+Bound Serializer::DecodeBound(google_firestore_v1_Cursor& cursor,
+                              bool as_start) const {
```

Comment:
Same - maybe we should pass in the cursor values and `inclusive`

Reply:
Done..more or less.